packages/dappmanager/src/calls/fetchDnpRequest.ts

```diff
@@ -147,11 +152,28 @@ function getRequiresCoreUpdate(
     : "";
   return Boolean(
     manifest.requirements &&
+      minDnVersion &&
       valid(minDnVersion) &&
       valid(coreVersion) &&
       gt(minDnVersion, coreVersion)
   );
 }
+async function getRequiresDockerUpdate({
+  manifest
+}: {
+  manifest: Manifest;
+}): Promise<boolean> {
+  const minDockerVersion = manifest.requirements?.minimumDockerVersion;
+  if (!minDockerVersion) return false;
+  const currentDockerVersion = await getDockerVersion();
+  return Boolean(
+    manifest.requirements &&
+      minDockerVersion &&
```

Comment:
the `manifest.requirements &&` can be removed since `minDockerVersion` already checks that

Reply:
done